scripts/sessionx.sh

```diff
@@ -169,7 +169,7 @@ handle_args() {
         --layout="$layout_mode" \
         --pointer=$pointer_icon \
         -p "$window_width,$window_height" \
-        --prompt $prompt_icon \
+        --prompt "$prompt_icon" \
```

Comment:
did you test this? can share a print?

Doesn't work for me:
![image](https://github.com/omerxx/tmux-sessionx/assets/108758883/cc38d66c-1ccc-478c-9b14-47663c959173)

Reply:
I see, needs to set a space in the configuration:
 
```bash
set -g @sessionx-prompt " "
set -g @sessionx-pointer "▶ "
```